client/orderbook/orderbook.go

```diff
@@ -24,6 +25,15 @@ type Order struct {
 	Epoch uint64
 }
 
+// copy creates a copy. Note that the OrderID is not a deep copy.
+func (o *Order) copy() *Order {
+	return &(*o)
```

Comment:
Since order.OrderID is an array, it would be a deep copy too.

Reply:
Oh, yeah. Nice.

Reply:
Comment still says `not a deep copy`

Reply:
crap

Reply:
That's alright.  I realized this wasn't making a copy of anything actually.  Go was simplifying it to just returning the same pointer: https://github.com/decred/dcrdex/commit/a71b9d6724af4a33c266beba7a5f6810fcf14618
Working on some tweaks for this.